### DISCUSSION TITLE
Local development setup redirects me to wiki.outline.dev after loading app.outline.dev:3000

### DISCUSSION THREAD

**Discussion by alkanna** [outline/outline#5965] [General] [answered]
I followed the local development setup procedure, and I used the `build/server/scripts/seed.js` script to seed a local user. However for some reason, after loading the app, it immediately loads a 301 to https://wiki.outline.dev.

Anyone know what might cause that ?

**Comment by tommoor** [comment]
Local development is optimized to work like the  cloud version currently, so each team has its own subdomain and uses the outline.dev tld.

This probably isn’t going to change as 99% of development is done by General Outline employees.

You just need to create an entry in /etc/hosts for the additional subdomain

**Reply by alkanna** [reply]
Hey, I did all of that. app.outline.dev:3000 properly displays the app, it's only afterwards that I am being redirected to wiki.outline.dev. I removed the subdomain from the workspace settings and it seems to have fixed it somehow.

**Comment by ktmouk** [comment, marked as answer]
`seed.js` appears to create a team with a wiki subdomain.
https://github.com/outline/outline/blob/74722b80f2b71703340297ceb2c36104bcd2ca87/server/scripts/seed.ts#L14-L15

we might need to add `wiki.outline.dev` to `/etc/hosts`.
```
127.0.0.1     app.outline.dev
127.0.0.1     wiki.outline.dev
```

I think updating this step on the [Local Development](https://docs.getoutline.com/s/hosting/doc/local-development-5hEhFRXow7) page would be beneficial for contributors.

**Reply by tommoor** [reply]
Updated
